src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$day-width: 100px;
+$day-height: 100px;
+$gap: 1px;
+$padding: 10px;
+
+$calendar-width: ($day-width * 7) + ($gap * 6) + ($padding * 2);
+
+
+
+
```

Comment:
So much empty lines :))
```suggestion

```

Reply:
True, don't know how lint didn't catch it :D